client/scss/main.scss

```diff
@@ -71,3 +71,31 @@ form button[type="submit"] {
   height: 30px;
   vertical-align: middle;
 }
+.header {
+    background-color: #2e80a9;
+    height: 20%;
+    text-align: center;
+}
+
+.text-input {
+    margin :20px;
+    .mdl-textfield{
+        width: 100% !important;
```

Comment:
do we really need important here?

Reply:
yes, doesnt work without it.

Reply:
thats weird, for me it does work perfectly without that

Reply:
you were right . I also changed some structure